ImageFeed/Auth/AuthViewController.swift

```diff
@@ -30,8 +30,15 @@ final class AuthViewController: UIViewController, WebViewViewControllerDelegate
         oauth.fetchAuthToken(code: code, completion: { [weak self] result in
             guard let self = self else { return }
             switch result {
-            case .failure(let error):
-                print(error.localizedDescription)
+            case .failure(_):
```

Comment:
Можно опускать параметры
```suggestion
            case .failure:
```